etcd/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "0.1.0",
```

Comment:
OK, then my comment on the apache manifest is on the wrong line, "manifest_version": 1` should be removed instead.